st2actions/st2actions/query/mistral/v2.py

```diff
@@ -115,9 +113,13 @@ def _get_workflow_result(self, exec_id):
         """
         execution = executions.ExecutionManager(self._client).get(exec_id)
 
-        output = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else None
+        result = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else {}
+        result['workflow'] = {
```

Comment:
This attribute is now Mistral workflow specific, right?

I'm fine with that change, maybe we should also make this a convention and put all the Mistral workflow specific stuff / data in the `workflow` attribute (in the past, we did our best to unify the result output for action chain and Mistral workflows, but there will always be cases where we can't do that).